[2025-07-29 13:37:19,414: INFO: main: Logging has been set up successfully.]
[2025-07-29 15:20:40,750: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 15:32:04,130: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 15:32:04,150: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 15:33:03,140: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 15:33:03,140: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 15:34:38,815: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 15:34:38,816: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 15:35:19,788: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 15:35:19,790: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 15:35:19,791: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 15:35:19,792: INFO: common: created directory at: artifacts]
[2025-07-29 15:35:19,793: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-29 15:35:20,610: INFO: 251335794: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: C80E:136ADB:E75AC:2290FB:68889CDC
Accept-Ranges: bytes
Date: Tue, 29 Jul 2025 10:05:17 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10230-MAA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1753783517.259678,VS0,VE268
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 1fb7901cc6b840d6644e154cf0e715386e20c830
Expires: Tue, 29 Jul 2025 10:10:17 GMT
Source-Age: 0

]
[2025-07-29 15:51:17,084: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-29 15:51:17,084: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 15:51:17,085: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 15:51:17,086: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 15:51:17,086: INFO: common: created directory at: artifacts]
[2025-07-29 15:51:17,086: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-29 15:51:17,881: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: C80E:136ADB:E75AC:2290FB:68889CDC
Accept-Ranges: bytes
Date: Tue, 29 Jul 2025 10:21:14 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10240-MAA
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1753784475.500439,VS0,VE270
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: c11ec90821ce733ee3b0a11fa4dcd4648ecc6100
Expires: Tue, 29 Jul 2025 10:26:14 GMT
Source-Age: 0

]
[2025-07-29 15:51:17,886: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-07-29 18:43:35,950: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 18:43:35,955: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 18:43:35,957: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 18:43:35,958: INFO: common: created directory at: artifacts]
[2025-07-29 18:45:28,098: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 18:45:28,100: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 18:45:28,101: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 18:45:28,101: INFO: common: created directory at: artifacts]
[2025-07-29 18:45:28,102: INFO: common: created directory at: artifacts/data_validation]
[2025-07-29 18:50:14,404: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-29 18:50:14,405: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 18:50:14,407: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 18:50:14,408: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 18:50:14,408: INFO: common: created directory at: artifacts]
[2025-07-29 18:50:14,408: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-29 18:50:14,408: INFO: data_ingestion: File already exists]
[2025-07-29 18:50:14,410: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-07-29 18:50:14,410: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2025-07-29 18:50:14,410: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 18:50:14,410: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 18:50:14,411: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 18:50:14,411: INFO: common: created directory at: artifacts]
[2025-07-29 18:50:14,411: INFO: common: created directory at: artifacts/data_validation]
[2025-07-29 18:50:14,415: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2025-07-29 19:13:18,838: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 19:13:18,843: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 19:13:18,845: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 19:13:18,846: INFO: common: created directory at: artifacts]
[2025-07-29 19:13:18,846: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-29 19:13:18,937: INFO: 741865018: Splited data into training and test sets]
[2025-07-29 19:13:18,938: INFO: 741865018: (1199, 12)]
[2025-07-29 19:13:18,938: INFO: 741865018: (400, 12)]
[2025-07-29 19:17:15,454: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-29 19:17:15,455: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 19:17:15,456: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 19:17:15,456: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 19:17:15,456: INFO: common: created directory at: artifacts]
[2025-07-29 19:17:15,456: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-29 19:17:15,456: INFO: data_ingestion: File already exists]
[2025-07-29 19:17:15,458: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-07-29 19:17:15,459: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2025-07-29 19:17:15,459: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 19:17:15,459: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 19:17:15,460: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 19:17:15,460: INFO: common: created directory at: artifacts]
[2025-07-29 19:17:15,460: INFO: common: created directory at: artifacts/data_validation]
[2025-07-29 19:17:15,465: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2025-07-29 19:17:15,465: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2025-07-29 19:17:15,466: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 19:17:15,466: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 19:17:15,466: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 19:17:15,466: INFO: common: created directory at: artifacts]
[2025-07-29 19:17:15,467: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-29 19:17:15,467: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2025-07-29 19:20:39,317: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-29 19:20:39,318: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 19:20:39,318: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 19:20:39,319: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 19:20:39,319: INFO: common: created directory at: artifacts]
[2025-07-29 19:20:39,319: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-29 19:20:39,319: INFO: data_ingestion: File already exists]
[2025-07-29 19:20:39,320: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-07-29 19:20:39,320: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2025-07-29 19:20:39,321: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 19:20:39,321: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 19:20:39,321: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 19:20:39,321: INFO: common: created directory at: artifacts]
[2025-07-29 19:20:39,321: INFO: common: created directory at: artifacts/data_validation]
[2025-07-29 19:20:39,327: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2025-07-29 19:20:39,327: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2025-07-29 19:20:39,328: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 19:20:39,328: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 19:20:39,328: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 19:20:39,328: INFO: common: created directory at: artifacts]
[2025-07-29 19:20:39,328: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-29 19:20:39,328: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2025-07-29 19:20:52,956: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 19:20:52,956: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 19:20:52,957: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 19:20:52,958: INFO: common: created directory at: artifacts]
[2025-07-29 19:20:52,958: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-29 19:20:52,975: INFO: 741865018: Splited data into training and test sets]
[2025-07-29 19:20:52,975: INFO: 741865018: (1199, 12)]
[2025-07-29 19:20:52,976: INFO: 741865018: (400, 12)]
[2025-07-29 19:38:46,636: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 19:38:46,640: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 19:38:46,641: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 19:38:46,642: INFO: common: created directory at: artifacts]
[2025-07-29 19:38:46,642: INFO: common: created directory at: artifacts/model_trainer]
[2025-07-29 19:42:19,547: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-29 19:42:19,548: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 19:42:19,549: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 19:42:19,550: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 19:42:19,550: INFO: common: created directory at: artifacts]
[2025-07-29 19:42:19,550: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-29 19:42:19,550: INFO: data_ingestion: File already exists]
[2025-07-29 19:42:19,551: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-07-29 19:42:19,551: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2025-07-29 19:42:19,552: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 19:42:19,552: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 19:42:19,553: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 19:42:19,553: INFO: common: created directory at: artifacts]
[2025-07-29 19:42:19,553: INFO: common: created directory at: artifacts/data_validation]
[2025-07-29 19:42:19,555: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2025-07-29 19:42:19,555: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2025-07-29 19:42:19,556: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 19:42:19,556: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 19:42:19,557: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 19:42:19,557: INFO: common: created directory at: artifacts]
[2025-07-29 19:42:19,557: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-29 19:42:19,557: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2025-07-29 19:42:19,557: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2025-07-29 19:42:19,558: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 19:42:19,558: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 19:42:19,558: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 19:42:19,558: INFO: common: created directory at: artifacts]
[2025-07-29 19:42:19,558: INFO: common: created directory at: artifacts/model_trainer]
[2025-07-29 19:42:19,564: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2025-07-29 20:13:20,300: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:13:20,304: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:13:20,305: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:13:20,306: INFO: common: created directory at: artifacts]
[2025-07-29 20:13:20,307: INFO: common: created directory at: artifacts/model_evaluation]
[2025-07-29 20:13:20,484: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2025-07-29 20:21:37,825: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-29 20:21:37,826: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:21:37,828: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:21:37,828: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:21:37,829: INFO: common: created directory at: artifacts]
[2025-07-29 20:21:37,829: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-29 20:21:37,829: INFO: data_ingestion: File already exists]
[2025-07-29 20:21:37,830: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-07-29 20:21:37,830: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2025-07-29 20:21:37,831: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:21:37,831: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:21:37,832: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:21:37,832: INFO: common: created directory at: artifacts]
[2025-07-29 20:21:37,832: INFO: common: created directory at: artifacts/data_validation]
[2025-07-29 20:21:37,837: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2025-07-29 20:21:37,837: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2025-07-29 20:21:37,838: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:21:37,838: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:21:37,838: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:21:37,839: INFO: common: created directory at: artifacts]
[2025-07-29 20:21:37,839: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-29 20:21:37,839: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2025-07-29 20:21:37,839: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2025-07-29 20:21:37,839: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:21:37,840: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:21:37,840: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:21:37,840: INFO: common: created directory at: artifacts]
[2025-07-29 20:21:37,840: INFO: common: created directory at: artifacts/model_trainer]
[2025-07-29 20:21:37,849: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2025-07-29 20:21:37,849: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<]
[2025-07-29 20:21:37,850: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:21:37,850: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:21:37,851: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:21:37,851: INFO: common: created directory at: artifacts]
[2025-07-29 20:21:37,851: INFO: common: created directory at: artifacts/model_evaluation]
[2025-07-29 20:21:38,537: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2025-07-29 20:21:44,314: ERROR: main: INTERNAL_ERROR: Response: {'error': 'unsupported endpoint, please contact support@dagshub.com'}]
Traceback (most recent call last):
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/main.py", line 53, in <module>
    data_ingestion.initiate_model_evaluation()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/src/datascience/pipeline/model_evaluation_pipeline.py", line 15, in initiate_model_evaluation
    model_evaluation.log_into_mlflow()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/src/datascience/components/model_evaluation.py", line 66, in log_into_mlflow
    mlflow.sklearn.log_model(model, "model", registered_model_name="ElasticnetModel")
    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/sklearn/__init__.py", line 426, in log_model
    return Model.log(
           ~~~~~~~~~^
        artifact_path=artifact_path,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<18 lines>...
        model_id=model_id,
        ^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/models/model.py", line 1161, in log
    model = mlflow.initialize_logged_model(
        # TODO: Update model name
    ...<6 lines>...
        else None,
    )
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/fluent.py", line 2130, in initialize_logged_model
    model = _create_logged_model(
        name=name,
    ...<4 lines>...
        experiment_id=experiment_id,
    )
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/fluent.py", line 2257, in _create_logged_model
    return MlflowClient().create_logged_model(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        experiment_id=experiment_id,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<4 lines>...
        model_type=model_type,
        ^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/client.py", line 5369, in create_logged_model
    return self._tracking_client.create_logged_model(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        experiment_id, name, source_run_id, tags, params, model_type
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/_tracking_service/client.py", line 824, in create_logged_model
    return self.store.create_logged_model(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        experiment_id=experiment_id,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<8 lines>...
        model_type=model_type,
        ^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/store/tracking/rest_store.py", line 936, in create_logged_model
    response_proto = self._call_endpoint(CreateLoggedModel, req_body)
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/store/tracking/rest_store.py", line 135, in _call_endpoint
    return call_endpoint(
        self.get_host_creds(),
    ...<4 lines>...
        retry_timeout_seconds=retry_timeout_seconds,
    )
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/utils/rest_utils.py", line 594, in call_endpoint
    response = verify_rest_response(response, endpoint)
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/utils/rest_utils.py", line 308, in verify_rest_response
    raise RestException(json.loads(response.text))
mlflow.exceptions.RestException: INTERNAL_ERROR: Response: {'error': 'unsupported endpoint, please contact support@dagshub.com'}
[2025-07-29 20:23:27,868: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-29 20:23:27,869: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:23:27,869: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:23:27,870: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:23:27,870: INFO: common: created directory at: artifacts]
[2025-07-29 20:23:27,870: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-29 20:23:27,870: INFO: data_ingestion: File already exists]
[2025-07-29 20:23:27,871: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-07-29 20:23:27,871: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2025-07-29 20:23:27,872: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:23:27,872: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:23:27,873: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:23:27,873: INFO: common: created directory at: artifacts]
[2025-07-29 20:23:27,873: INFO: common: created directory at: artifacts/data_validation]
[2025-07-29 20:23:27,876: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2025-07-29 20:23:27,876: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2025-07-29 20:23:27,876: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:23:27,877: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:23:27,877: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:23:27,877: INFO: common: created directory at: artifacts]
[2025-07-29 20:23:27,877: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-29 20:23:27,877: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2025-07-29 20:23:27,877: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2025-07-29 20:23:27,878: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:23:27,878: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:23:27,879: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:23:27,879: INFO: common: created directory at: artifacts]
[2025-07-29 20:23:27,879: INFO: common: created directory at: artifacts/model_trainer]
[2025-07-29 20:23:27,883: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2025-07-29 20:23:27,883: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<]
[2025-07-29 20:23:27,884: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:23:27,884: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:23:27,885: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:23:27,885: INFO: common: created directory at: artifacts]
[2025-07-29 20:23:27,885: INFO: common: created directory at: artifacts/model_evaluation]
[2025-07-29 20:23:28,537: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2025-07-29 20:23:35,785: ERROR: main: INTERNAL_ERROR: Response: {'error': 'unsupported endpoint, please contact support@dagshub.com'}]
Traceback (most recent call last):
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/main.py", line 53, in <module>
    data_ingestion.initiate_model_evaluation()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/src/datascience/pipeline/model_evaluation_pipeline.py", line 15, in initiate_model_evaluation
    model_evaluation.log_into_mlflow()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/src/datascience/components/model_evaluation.py", line 66, in log_into_mlflow
    mlflow.sklearn.log_model(model, "model")
    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/sklearn/__init__.py", line 426, in log_model
    return Model.log(
           ~~~~~~~~~^
        artifact_path=artifact_path,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<18 lines>...
        model_id=model_id,
        ^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/models/model.py", line 1161, in log
    model = mlflow.initialize_logged_model(
        # TODO: Update model name
    ...<6 lines>...
        else None,
    )
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/fluent.py", line 2130, in initialize_logged_model
    model = _create_logged_model(
        name=name,
    ...<4 lines>...
        experiment_id=experiment_id,
    )
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/fluent.py", line 2257, in _create_logged_model
    return MlflowClient().create_logged_model(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        experiment_id=experiment_id,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<4 lines>...
        model_type=model_type,
        ^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/client.py", line 5369, in create_logged_model
    return self._tracking_client.create_logged_model(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        experiment_id, name, source_run_id, tags, params, model_type
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/_tracking_service/client.py", line 824, in create_logged_model
    return self.store.create_logged_model(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        experiment_id=experiment_id,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<8 lines>...
        model_type=model_type,
        ^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/store/tracking/rest_store.py", line 936, in create_logged_model
    response_proto = self._call_endpoint(CreateLoggedModel, req_body)
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/store/tracking/rest_store.py", line 135, in _call_endpoint
    return call_endpoint(
        self.get_host_creds(),
    ...<4 lines>...
        retry_timeout_seconds=retry_timeout_seconds,
    )
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/utils/rest_utils.py", line 594, in call_endpoint
    response = verify_rest_response(response, endpoint)
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/utils/rest_utils.py", line 308, in verify_rest_response
    raise RestException(json.loads(response.text))
mlflow.exceptions.RestException: INTERNAL_ERROR: Response: {'error': 'unsupported endpoint, please contact support@dagshub.com'}
[2025-07-29 20:42:55,991: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.151:8080]
[2025-07-29 20:42:55,992: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-07-29 20:43:06,413: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:06] "GET / HTTP/1.1" 200 -]
[2025-07-29 20:43:06,561: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:06] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-07-29 20:43:06,724: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:06] "[33mGET /apple-touch-icon-precomposed.png HTTP/1.1[0m" 404 -]
[2025-07-29 20:43:06,972: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:06] "[33mGET /apple-touch-icon.png HTTP/1.1[0m" 404 -]
[2025-07-29 20:43:07,836: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:07] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2025-07-29 20:43:10,981: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:10] "GET / HTTP/1.1" 200 -]
[2025-07-29 20:43:10,988: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:10] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-07-29 20:43:16,532: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-29 20:43:16,534: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:43:16,535: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:43:16,536: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:43:16,536: INFO: common: created directory at: artifacts]
[2025-07-29 20:43:16,536: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-29 20:43:16,536: INFO: data_ingestion: File already exists]
[2025-07-29 20:43:16,538: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-07-29 20:43:16,538: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2025-07-29 20:43:16,539: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:43:16,539: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:43:16,540: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:43:16,540: INFO: common: created directory at: artifacts]
[2025-07-29 20:43:16,540: INFO: common: created directory at: artifacts/data_validation]
[2025-07-29 20:43:16,546: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2025-07-29 20:43:16,546: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2025-07-29 20:43:16,547: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:43:16,547: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:43:16,548: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:43:16,548: INFO: common: created directory at: artifacts]
[2025-07-29 20:43:16,548: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-29 20:43:16,548: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2025-07-29 20:43:16,548: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2025-07-29 20:43:16,548: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:43:16,549: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:43:16,549: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:43:16,549: INFO: common: created directory at: artifacts]
[2025-07-29 20:43:16,549: INFO: common: created directory at: artifacts/model_trainer]
[2025-07-29 20:43:16,561: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2025-07-29 20:43:16,561: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<]
[2025-07-29 20:43:16,562: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:43:16,563: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:43:16,563: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:43:16,563: INFO: common: created directory at: artifacts]
[2025-07-29 20:43:16,563: INFO: common: created directory at: artifacts/model_evaluation]
[2025-07-29 20:43:17,465: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2025-07-29 20:43:22,580: ERROR: main: INTERNAL_ERROR: Response: {'error': 'unsupported endpoint, please contact support@dagshub.com'}]
Traceback (most recent call last):
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/main.py", line 53, in <module>
    data_ingestion.initiate_model_evaluation()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/src/datascience/pipeline/model_evaluation_pipeline.py", line 15, in initiate_model_evaluation
    model_evaluation.log_into_mlflow()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/src/datascience/components/model_evaluation.py", line 66, in log_into_mlflow
    mlflow.sklearn.log_model(model, "model")
    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/sklearn/__init__.py", line 426, in log_model
    return Model.log(
           ~~~~~~~~~^
        artifact_path=artifact_path,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<18 lines>...
        model_id=model_id,
        ^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/models/model.py", line 1161, in log
    model = mlflow.initialize_logged_model(
        # TODO: Update model name
    ...<6 lines>...
        else None,
    )
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/fluent.py", line 2130, in initialize_logged_model
    model = _create_logged_model(
        name=name,
    ...<4 lines>...
        experiment_id=experiment_id,
    )
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/fluent.py", line 2257, in _create_logged_model
    return MlflowClient().create_logged_model(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        experiment_id=experiment_id,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<4 lines>...
        model_type=model_type,
        ^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/client.py", line 5369, in create_logged_model
    return self._tracking_client.create_logged_model(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        experiment_id, name, source_run_id, tags, params, model_type
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/_tracking_service/client.py", line 824, in create_logged_model
    return self.store.create_logged_model(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        experiment_id=experiment_id,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<8 lines>...
        model_type=model_type,
        ^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/store/tracking/rest_store.py", line 936, in create_logged_model
    response_proto = self._call_endpoint(CreateLoggedModel, req_body)
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/store/tracking/rest_store.py", line 135, in _call_endpoint
    return call_endpoint(
        self.get_host_creds(),
    ...<4 lines>...
        retry_timeout_seconds=retry_timeout_seconds,
    )
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/utils/rest_utils.py", line 594, in call_endpoint
    response = verify_rest_response(response, endpoint)
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/utils/rest_utils.py", line 308, in verify_rest_response
    raise RestException(json.loads(response.text))
mlflow.exceptions.RestException: INTERNAL_ERROR: Response: {'error': 'unsupported endpoint, please contact support@dagshub.com'}
[2025-07-29 20:43:22,779: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:22] "GET /train HTTP/1.1" 200 -]
[2025-07-29 20:43:36,393: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:36] "GET / HTTP/1.1" 200 -]
[2025-07-29 20:43:36,397: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:36] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-07-29 20:43:37,298: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-29 20:43:37,299: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:43:37,299: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:43:37,300: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:43:37,300: INFO: common: created directory at: artifacts]
[2025-07-29 20:43:37,300: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-29 20:43:37,300: INFO: data_ingestion: File already exists]
[2025-07-29 20:43:37,301: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-07-29 20:43:37,301: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2025-07-29 20:43:37,302: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:43:37,302: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:43:37,303: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:43:37,303: INFO: common: created directory at: artifacts]
[2025-07-29 20:43:37,303: INFO: common: created directory at: artifacts/data_validation]
[2025-07-29 20:43:37,306: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2025-07-29 20:43:37,306: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2025-07-29 20:43:37,306: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:43:37,307: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:43:37,307: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:43:37,307: INFO: common: created directory at: artifacts]
[2025-07-29 20:43:37,307: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-29 20:43:37,307: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2025-07-29 20:43:37,307: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2025-07-29 20:43:37,308: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:43:37,308: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:43:37,309: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:43:37,309: INFO: common: created directory at: artifacts]
[2025-07-29 20:43:37,309: INFO: common: created directory at: artifacts/model_trainer]
[2025-07-29 20:43:37,314: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2025-07-29 20:43:37,314: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<]
[2025-07-29 20:43:37,315: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/config/config.yaml loaded successfully]
[2025-07-29 20:43:37,315: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/params.yaml loaded successfully]
[2025-07-29 20:43:37,315: INFO: common: yaml file: /Users/lokesh/Desktop/MLFLOW/datascienceproject/schema.yaml loaded successfully]
[2025-07-29 20:43:37,315: INFO: common: created directory at: artifacts]
[2025-07-29 20:43:37,315: INFO: common: created directory at: artifacts/model_evaluation]
[2025-07-29 20:43:37,972: INFO: common: json file saved at: artifacts/model_evaluation/metrics.json]
[2025-07-29 20:43:38,071: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:38] "GET / HTTP/1.1" 200 -]
[2025-07-29 20:43:38,078: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:38] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-07-29 20:43:40,391: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:40] "GET /predict HTTP/1.1" 200 -]
[2025-07-29 20:43:40,397: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:40] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-07-29 20:43:42,941: ERROR: main: INTERNAL_ERROR: Response: {'error': 'unsupported endpoint, please contact support@dagshub.com'}]
Traceback (most recent call last):
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/main.py", line 53, in <module>
    data_ingestion.initiate_model_evaluation()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/src/datascience/pipeline/model_evaluation_pipeline.py", line 15, in initiate_model_evaluation
    model_evaluation.log_into_mlflow()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/src/datascience/components/model_evaluation.py", line 66, in log_into_mlflow
    mlflow.sklearn.log_model(model, "model")
    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/sklearn/__init__.py", line 426, in log_model
    return Model.log(
           ~~~~~~~~~^
        artifact_path=artifact_path,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<18 lines>...
        model_id=model_id,
        ^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/models/model.py", line 1161, in log
    model = mlflow.initialize_logged_model(
        # TODO: Update model name
    ...<6 lines>...
        else None,
    )
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/fluent.py", line 2130, in initialize_logged_model
    model = _create_logged_model(
        name=name,
    ...<4 lines>...
        experiment_id=experiment_id,
    )
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/fluent.py", line 2257, in _create_logged_model
    return MlflowClient().create_logged_model(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        experiment_id=experiment_id,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<4 lines>...
        model_type=model_type,
        ^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/client.py", line 5369, in create_logged_model
    return self._tracking_client.create_logged_model(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        experiment_id, name, source_run_id, tags, params, model_type
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/tracking/_tracking_service/client.py", line 824, in create_logged_model
    return self.store.create_logged_model(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        experiment_id=experiment_id,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<8 lines>...
        model_type=model_type,
        ^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/store/tracking/rest_store.py", line 936, in create_logged_model
    response_proto = self._call_endpoint(CreateLoggedModel, req_body)
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/store/tracking/rest_store.py", line 135, in _call_endpoint
    return call_endpoint(
        self.get_host_creds(),
    ...<4 lines>...
        retry_timeout_seconds=retry_timeout_seconds,
    )
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/utils/rest_utils.py", line 594, in call_endpoint
    response = verify_rest_response(response, endpoint)
  File "/Users/lokesh/Desktop/MLFLOW/datascienceproject/venv/lib/python3.13/site-packages/mlflow/utils/rest_utils.py", line 308, in verify_rest_response
    raise RestException(json.loads(response.text))
mlflow.exceptions.RestException: INTERNAL_ERROR: Response: {'error': 'unsupported endpoint, please contact support@dagshub.com'}
[2025-07-29 20:43:43,160: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:43:43] "GET /train HTTP/1.1" 200 -]
[2025-07-29 20:44:05,151: INFO: _internal: 127.0.0.1 - - [29/Jul/2025 20:44:05] "POST /predict HTTP/1.1" 200 -]
